hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -23,80 +23,95 @@
 import org.apache.hadoop.hdds.conf.ConfigGroup;
 import org.apache.hadoop.hdds.conf.ConfigTag;
 import org.apache.hadoop.hdds.conf.ConfigType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * This class contains configuration values for the ContainerBalancer.
  */
 @ConfigGroup(prefix = "hdds.container.balancer.")
 public final class ContainerBalancerConfiguration {
+  private static final Logger LOG =
+      LoggerFactory.getLogger(ContainerBalancerConfiguration.class);
+
   @Config(key = "utilization.threshold", type = ConfigType.AUTO, defaultValue =
       "0.1", tags = {ConfigTag.BALANCER},
       description = "Threshold is a fraction in the range of 0 to 1. A " +
           "cluster is considered balanced if for each datanode, the " +
           "utilization of the datanode (used space to capacity ratio) differs" +
           " from the utilization of the cluster (used space to capacity ratio" +
           " of the entire cluster) no more than the threshold value.")
-  private double threshold = 0.1;
+  private String threshold = "0.1";
 
   @Config(key = "datanodes.balanced.max", type = ConfigType.INT,
       defaultValue = "5", tags = {ConfigTag.BALANCER}, description = "The " +
       "maximum number of datanodes that should be balanced. Container " +
       "Balancer will not balance more number of datanodes than this limit.")
   private int maxDatanodesToBalance = 5;
 
-  @Config(key = "size.moved.max", type = ConfigType.LONG,
-      defaultValue = "10737418240L", tags = {ConfigTag.BALANCER},
-      description = "The maximum size of data in Bytes that will be moved " +
-          "by the Container Balancer.")
-  private long maxSizeToMove = 10737418240L;
+  @Config(key = "size.moved.max", type = ConfigType.SIZE,
+      defaultValue = "10", tags = {ConfigTag.BALANCER},
```

Comment:
Please see how ConfigType.SIZE is used in other places. I think the default value should be `10GB`. It is more flexible config type and it will not be limited to GB alone. The description and javadoc for corresponding fns can be updated to remove specificity to GB.